build-support/gen_version_info.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This could be a problem for tp2 builds. Updating tp2 deps to force pyton3 resolves it for now